src/pages/MyKiva/MyKivaPageContent.vue

```diff
@@ -77,6 +77,7 @@
 					:lender="lender"
 					:total-updates="totalUpdates"
 					:updates-loading="updatesLoading"
+					:balance="newBalance"
```

Comment:
The `newBalance` isn't defined on the component itself, so you'll need to source the balance in a different way, for example the component does have `userBalance`.

Reply:
Actually, you should be able to set the balance on the repayment data in `formatRepaymentCards`, since we already use balance in that method.